apps/web/src/pages/invites/MembersInvitePage.tsx

```diff
@@ -0,0 +1,105 @@
+import { Form } from 'antd';
+import { useMutation, useQuery } from 'react-query';
+import styled from 'styled-components';
+import { MemberStatusEnum } from '@notifire/shared';
+import * as capitalize from 'lodash.capitalize';
+import { Avatar, Container, Divider, Group, Text } from '@mantine/core';
+import PageHeader from '../../components/layout/components/PageHeader';
+import { getOrganizationMembers, inviteMember } from '../../api/organization';
+import PageContainer from '../../components/layout/components/PageContainer';
+import { Button, Input, Tag } from '../../design-system';
+import { Invite } from '../../design-system/icons';
+import useStyles from '../../design-system/config/text.styles';
+
+export function MembersInvitePage() {
+  const [form] = Form.useForm();
```

Comment:
As our new design system no longer uses antd, we try to use it as little as possible. And forms especially, because some are quite big. We are migrating all forms to use react-hook-form. You could see an example in [Settings](https://github.com/notifirehq/notifire/blob/13f6433647ec829b47332bf4fce2669672f355fc/apps/web/src/pages/settings/components/EmailSettingsForm.tsx) page
Would you mind taking a look? :)